ui/packages/components/src/RunDetailsV3/RunInfo.tsx

```diff
@@ -87,7 +87,7 @@ export const RunInfo = ({
             onClick={() => setExpanded(!expanded)}
           >
             <RiArrowRightSLine
-              className={`shrink-0 transition-transform duration-500 ${
+              className={`shrink-0 transition-transform duration-[250ms] ${
```

Comment:
Thankfully tailwind 4 will allow to define the values, so it will accept duration-250